src/chartjs/chartjs.directive.js

```diff
@@ -1,5 +1,5 @@
 import _ from 'lodash';
-import Chart from 'chart';
+import Chart from 'chart.js'; // eslint-disable-line
```

Comment:
What rule are you disabling ?

Reply:
Due to the package name (ref: https://github.com/chartjs/Chart.js/blob/master/package.json#L2)